dao-services/src/main/java/org/exoplatform/wiki/liquibase/TimestampChange.java

```diff
@@ -0,0 +1,82 @@
+package org.exoplatform.wiki.liquibase;
+
+import liquibase.change.custom.CustomSqlChange;
+import liquibase.database.Database;
+import liquibase.exception.CustomChangeException;
+import liquibase.exception.SetupException;
+import liquibase.exception.ValidationErrors;
+import liquibase.resource.ResourceAccessor;
+import liquibase.statement.SqlStatement;
+import liquibase.statement.core.ModifyDataTypeStatement;
+
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+/**
+ * Custom tag Liquibase implements CustomSqlChange
+ * This service will be used to update Default TimeStamp.
```

Comment:
Can you precise that we needed a custom change to be able to set correctly the UTC time please ?